mozilla/bedrock
# [fix bug 1293539] Decommission and redirect Firefox Tour

## Description
- Decomissions the old Firefox tour and redirects /firefox/tour/ URL to SUMO.

## Bugzilla link
https://bugzilla.mozilla.org/show_bug.cgi?id=1293539

## Testing
- Check to make sure I've not deleted anything that's still needed, or left anything that can now also be deleted.
- Check that redirect works as expected.

## Checklist
